<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QFrame" name="frameMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>640</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>ПОСТРОЕНИЕ ЗАМЫКАНИЯ АТРИБУТА</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>760</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAction">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Построить замыкание</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataBase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>База Данных:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAtribute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атрибут:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAtribute">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>220</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Подключить БД</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
